types/index.d.ts

```diff
@@ -0,0 +1,44 @@
+// Type definitions for postcss-load-config 2.1
```

Comment:
Do you copy it from `@types/postcss-load-config`. If yes, we need to mention the original project here.

Reply:
done